modules/restarting-the-central-container.adoc

```diff
@@ -2,30 +2,32 @@
 //
 // * configuration/add-trusted-ca.adoc
 // * configuration/configure-endpoints.adoc
+// * configuration/add-custom-certificates.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="restart-central_{context}"]
+[id="restarting-the-central-container_{context}"]
 = Restarting the Central container
 
 [role="_abstract"]
-You can restart the Central container by killing the Central container or by deleting the Central pod.
+You can restart the Central container by killing or by deleting the Central pod.
 
 .Procedure
 
-* Run the following command to kill the Central container:
-+
-[NOTE]
-====
-You must wait for at least 1 minute, until {ocp} propagates your changes and restarts the Central container.
-====
+* To kill the Central container, run the following command:
```

Comment:
You changed the wording in the abstract to say "killing or deleting the Central pod" so should it be "kill the Central pod" here?

Reply:
Thanks for noticing this @kcarmichael08 . We have to delete the pods and not restart the containers:
- the `oc` call below shows a pod deletion, not killing the container
- secondly, killing the containers wouldn't even work for TLS cert reloading

So we should remove any mentions of killing containers in this context.